packages/editor-ui/src/components/MainSidebarSourceControl.vue

```diff
@@ -36,8 +34,11 @@ const tooltipOpenDelay = ref(300);
 const currentBranch = computed(() => {
 	return sourceControlStore.preferences.branchName;
 });
-const isInstanceOwner = computed(() => usersStore.isInstanceOwner);
-const setupButtonTooltipPlacement = computed(() => (props.isCollapsed ? 'right' : 'top'));
+const sourceControlAvailable = computed(
+	() =>
+		sourceControlStore.isEnterpriseSourceControlEnabled &&
+		rbacStore.hasScope('sourceControl:manage'),
```

Comment:
Can you use `hasPermission(['rbac'], { rbac: { scope: 'sourceControl:manage' } })` to keep consistent?

Reply:
Done